Bucketeer/Sources/Internal/Evaluation/EvaluationStorageImpl.swift

```diff
@@ -0,0 +1,110 @@
+final class EvaluationStorageImpl: EvaluationStorage {
+
+    var currentEvaluationsId: String {
+        get {
+            return evaluationUserDefaultsDao.currentEvaluationsId
+        }
+        set {
+            evaluationUserDefaultsDao.currentEvaluationsId = newValue
+        }
+    }
+
+    var featureTag: String {
+        get {
+            return evaluationUserDefaultsDao.featureTag
+        }
+        set {
+            evaluationUserDefaultsDao.featureTag = newValue
+        }
+    }
+
+    var evaluatedAt: String {
+        get {
+            return evaluationUserDefaultsDao.evaluatedAt
+        }
+        set {
+            evaluationUserDefaultsDao.evaluatedAt = newValue
+        }
+    }
+
+    var userAttributesUpdated: Bool {
+        get {
+            return evaluationUserDefaultsDao.userAttributesUpdated
+        }
+        set {
+            evaluationUserDefaultsDao.userAttributesUpdated = newValue
+        }
+    }
+
+    private let userId: String
+    // Expected SQL Dao
+    private let evaluationDao: EvaluationDao
+    // Expected in-memory cache Dao
+    private let evaluationMemCacheDao: EvaluationMemCacheDao
+    // using `var` for mutable evaluationUserDefaultDao properties | `let` will return error "could not mutable the immutable object"
+    private var evaluationUserDefaultsDao: EvaluationUserDefaultsDao
```

Comment:
I prefer to declare dedicated getter/setter func rather than `var` properties and make this `let`, but not sure if it's swift way.

Reply:
I added some changes tobe able using `let` on this property